kit/Kit.cpp

```diff
@@ -1376,6 +1382,36 @@ void Document::handleSaveMessage(const std::string &)
     }
 }
 
+void Document::startSaveWatchdog()
+{
+    _bgSaveWatchdogThread = std::thread(
+        [this]()
+        {
+            Util::setThreadName("kitbgsv_" + Util::encodeId(_mobileAppDocId, 3) + "_wdg");
+
+            const auto timeout =
+                std::chrono::seconds(ConfigUtil::getInt("per_document.bgsave_timeout_secs", 3));
```

Comment:
Default here does not match the one in `coolwsd.xml.in` which has 30 seconds, really unimportant.

Reply:
Good catch! That was me debugging. Thanks.